calendars/conferences.yaml

```
@@ -5,6 +5,14 @@ description: |
   events
 
 events:
+  - summary: Nordic-RSE conference 2025
+    description: |
+      Nordic-RSE 2025 in person Conference. See the event page
+      at https://nordic-rse.org/nrse2025/
+    begin: 2025-05-20 
+    duration: { days: 2 }
```

From this example it shows begin and end days. I don't know if this works, but maybe it's safer to use the one form the example since I probably tested it at some point...

https://github.com/coderefinery/git-calendar/blob/5b11d423759d225e9766d8c961b545f5321df2bf/calendars/example.yaml#L32-L37
```suggestion
    begin: 2025-05-20 
    end: 2025-05-21
```